templates/docs/examples/base/variable-font-test.html

```diff
@@ -0,0 +1,46 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Variable font{% endblock %}
+
+{% block standalone_css %}base{% endblock %}
+
+
+
+
+{% block content %}
+<style>
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wdth,wght@0,75..100,300..800;1,75..100,300..800&display=swap');
+
+.mono{
```

Comment:
do we need custom class names?

I guess we can simply use built-in stuff, like `<em>` for italic and `<code>` for mono?